### PR TITLE
Update deps to 1.16

### PR DESCRIPTION
**Pull request by marun**

### CODE REVIEW

**Comment by marun** [comment]
Failing e2e jobs appear to be consistent, suggesting that this PR is the cause.

**Comment by sttts** [comment]
Seeing `remote error: tls: bad certificate` in the openshift-apiserver logs.

**Comment by marun** [comment]
Tests are passing, but PR needs to be held until https://github.com/openshift/library-go/pull/538 merges and the change is vendored in.

**Comment by marun** [comment]
@sttts Ready

File: pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go [hunk 1]
```diff
@@ -463,6 +463,7 @@ func manageAPIServices_v311_00_to_latest(client apiregistrationv1client.APIServi
 				VersionPriority:      15,
 			},
 		}
+		apiregistrationv1.SetDefaults_ServiceReference(obj.Spec.Service)
```
Comment (sttts) [inline pkg/operator/workloadcontroller/workload_controller_openshiftapiserver_v311_00.go]: oh, you are lucky here. Defaults are considered private to the API server. But we make them accessible by accident.

**Pull request merged**